launcher/commands/usvfs.py

```diff
@@ -50,3 +58,5 @@ def run(self, args) -> None:
 
         print('Reapplying Anomaly binary dir to install directory')
         copytree(anomaly_dir / 'bin', install_dir / 'bin', dirs_exist_ok=True)
+        if Path(install_dir / 'appdata' / 'user.ltx~').exists():
+            copy2(install_dir / 'appdata' / 'user.ltx~', install_dir / 'appdata' / 'user.ltx')
```

Comment:
If following the last comment of this file, just rewrite user.ltx with the previous content

```python
if content_user_ltx:
    Path(...).write_text(content_user_ltx)
```